src/python/doc/perslay.md

```diff
@@ -0,0 +1,135 @@
+
+  * if **persistence_weight** is "linear":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **coeff_init**         | Initializer of the coefficient of the linear weight function. It can be either a single value, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(1., 1.). | 
+    | **coeff_const**        | Boolean specifying if **coeff_init** is initialized with a value (True) or randomly with tensorflow (False). |
+ 
+  * if **persistence_weight** is "grid":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **grid_size**          | Grid size of the grid weight function. It is a tuple of integer values, such as (10,10). | 
+    | **grid_bnds**          | Grid boundaries of the grid weight function. It is a tuple containing two tuples, each containing the minimum and maximum values of each axis of the plane. Example: ((-0.01, 1.01), (-0.01, 1.01)). | 
+    | **grid_init**          | Initializer for the pixel values of the grid weight function. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(1., 1.).| 
+    | **grid_const**         | Boolean specifying if **grid_init** is initialized with an array (True) or randomly with tensorflow (False). |
+
+  * if **persistence_weight** is "gmix":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **gmix_num**           | Number of Gaussian functions of the mixture of Gaussians weight function. |
+    | **gmix_m_init**        | Initializer of the means of the mixture of Gaussians weight function. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(1., 1.). |
+    | **gmix_m_const**       | Boolean specifying if **gmix_m_init** is initialized with an array (True) or randomly with tensorflow (False). |
+    | **gmix_v_init**        | Initializer of the bandwidths of the mixture of Gaussians weight function. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(1., 1.). |
+    | **gmix_v_const**       | Boolean specifying if **gmix_v_init** is initialized with an array (True) or randomly with tensorflow (False). Used only if **persistence_weight** is "gmix". |
+
+Depending on what **layer** is, the following additional keys are requested:
+
+  * if **layer** is "pm":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **peq**                | Sequence of permutation equivariant operations, as defined in [the DeepSet article](). It is a list of tuples of the form (*dim*, *operation*). Each tuple defines a permutation equivariant function of dimension *dim* and second permutation operation *operation* (string, either "max", "min", "sum" or None). Second permutation operation is optional and is not applied if *operation* is set to None. Example: [(150, "max"), (75, None)]. |
```

Comment:
Similarly to previous comment from Marc, I feel the names could be more explicit here.